lib/react_client.dart

```diff
@@ -260,7 +297,7 @@ final ReactDartInteropStatics _dartInteropStatics = (() {
 
   /// Wrapper for [Component.getInitialState].
   Component initComponent(ReactComponent jsThis, ReactDartComponentInternal internal, InteropContextValue context,
-          ComponentStatics componentStatics) =>
+          ComponentStatics<Component> componentStatics) =>
```

Comment:
Sorry... I missed this before... since we're only going to be using `ComponentStatics` for `Component`, not `Component` _and_ `Component2`, there is no need for the `ComponentStatics` class to have generic parameters.